src/delete_booking.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+// set up session
```

Comment:
May try to put this in the auto linter (and I know it's already how the code's been written), but let's have no empty line between comments and the code it's commenting on.